.github/workflows/packages-build-windows-agent.yml

```diff
@@ -0,0 +1,175 @@
+name: Build Wazuh Agent Packages - Windows
+on:
+  workflow_dispatch:
+    inputs:
+      docker_image_tag:
+        description: |
+          Tag name of the Docker image to be downloaded.
+          Use 'developer' to set branch name as tag.
+          Use 'auto' to set branch version as tag.
+          Default is 'auto'.
+        required: false
+        default: 'auto'
+      revision:
+        description: |
+          Package revision (name and metadata)
+          Default is '0'.
+        required: false
+        default: '0'
+      is_stage:
+        description: |
+          Should set development/production nomenclature
+          True if package name should have production format.
+          False if package name should have developer format.
+          Default is 'false'.
+        type: boolean
+        required: false
+        default: false
```

Comment:
These lines can be safely removed as they are already set to false by default. Leaving a comment explaining this (as you've done) is sufficient.